client/rest-high-level/src/test/java/org/opensearch/client/CrudIT.java

```diff
@@ -456,25 +447,7 @@ public void testMultiGetWithTypes() throws IOException {
         highLevelClient().bulk(bulk, expectWarningsOnce(RestBulkAction.TYPES_DEPRECATION_MESSAGE));
         MultiGetRequest multiGetRequest = new MultiGetRequest();
         multiGetRequest.add("index", "id1");
-        multiGetRequest.add("index", "type", "id2");
-
-        MultiGetResponse response = execute(
-            multiGetRequest,
-            highLevelClient()::mget,
-            highLevelClient()::mgetAsync,
-            expectWarningsOnce(RestMultiGetAction.TYPES_DEPRECATION_MESSAGE)
-        );
-        assertEquals(2, response.getResponses().length);
-
-        GetResponse firstResponse = response.getResponses()[0].getResponse();
-        assertEquals("index", firstResponse.getIndex());
-        assertEquals("type", firstResponse.getType());
-        assertEquals("id1", firstResponse.getId());
-
-        GetResponse secondResponse = response.getResponses()[1].getResponse();
-        assertEquals("index", secondResponse.getIndex());
-        assertEquals("type", secondResponse.getType());
-        assertEquals("id2", secondResponse.getId());
+        multiGetRequest.add("index", "id2");
```

Comment:
Minor, but the test name  `testMultiGetWithTypes()` is not reflecting anymore what the test is doing, `testMultiGetWithIds()` ?

Reply:
@nknize ^